src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp

```diff
@@ -91,12 +90,19 @@ TildeTimingMonitor::TildeTimingMonitor()
   uint32_t index = 0;
   for (auto & pinfo : required_paths_map_.at(params_.mode)) {
     // Subscriber
-    const auto gen_sub = create_generic_subscription(
-      pinfo.topic, pinfo.mtype, qos,
-      [this, &pinfo](const std::shared_ptr<rclcpp::SerializedMessage> msg) {
-        TildeTimingMonitor::onGenTopic(msg, pinfo);
-      });
-    gen_sub_buffer_.push_back(gen_sub);
+    try {
+      const auto gen_sub = create_generic_subscription(
+        pinfo.topic, pinfo.mtype, qos,
+        [this, &pinfo](const std::shared_ptr<rclcpp::SerializedMessage> msg) {
+          TildeTimingMonitor::onGenTopic(msg, pinfo);
+        });
+      gen_sub_buffer_.push_back(gen_sub);
+    } catch (const rclcpp::exceptions::RCLError & e) {
+      RCLCPP_INFO(
+        get_logger(), "\n[Exception] %s %s [%s]", e.what(), pinfo.topic.c_str(),
+        pinfo.mtype.c_str());
+      exit(-1);
```

Comment:
```suggestion
        exit(-1);
```

Reply:
I think it was formatted because it exceeded 100 characters.